en/controllers/request-response.rst

```diff
@@ -68,12 +68,14 @@ Query String Parameters
 
 .. php:method:: query($name)
 
-Query string parameters can be read using
-:php:attr:`~Cake\\Network\\Request::$query`::
+Query string parameters can be read using the ``getQuery()`` method::
 
     // URL is /posts/index?page=1&sort=title
     $this->request->getQuery('page');
```

Comment:
I would write ``$page = $this->request->getQuery('page');``, this way it becomes even more clear. I think this is nov more important than before, as we now prefix all setter methods with a ``$response = ``.

Shall I open a separate PR after this gets merged?